kie-docker-ui-app/kie-docker-ui-webapp/src/main/java/org/kie/dockerui/backend/util/Timer.java

```diff
@@ -0,0 +1,26 @@
+package org.kie.dockerui.backend.util;
+
+public class Timer {
```

Comment:
There is very nice class in commons-lang3 which we could use: https://commons.apache.org/proper/commons-lang/javadocs/api-3.4/index.html?org/apache/commons/lang3/time/StopWatch.html.